pubsub/cloud-client/subscriber.py

```diff
@@ -72,16 +72,65 @@ def create_subscription(project_id, topic_name, subscription_name):
         project_id, subscription_name
     )
 
-    subscription = subscriber.create_subscription(
-        subscription_path, topic_path
-    )
+    subscription = subscriber.create_subscription(subscription_path, topic_path)
 
     print("Subscription created: {}".format(subscription))
 
     subscriber.close()
     # [END pubsub_create_pull_subscription]
 
 
+def create_subscription_with_dead_letter_topic(
+    project_id, topic_name, subscription_name, dead_letter_topic_name
+):
+    """Create a subscription with dead letter policy."""
+    # [START pubsub_dead_letter_create_subscription]
+    from google.cloud import pubsub_v1
+    from google.cloud.pubsub_v1.types import DeadLetterPolicy
+
+    # TODO project_id = "Your Google Cloud Project ID"
+    # This is an existing topic that the subscription with dead letter policy
```

Comment:
Please put these in something like the following format: 
```
# TODO(developer): This is an existing topic that the subscription with dead letter policy
# project_id = "your-project-id"
```

This way the developer can just uncomment the line and replace the value. Additionally, if any of the variables require a specific format, it's helpful to hint that using the string name.

Reply:
Done.